index.html

```diff
@@ -14,7 +14,7 @@
     <!-- Header -->
 
     <header class="header-container">
-        <img src="assets/images/logo.jpeg" / class="hvr-pulse-grow">
+        <img src="assets/images/logo.jpeg" class="hvr-pulse-grow">
```

Comment:
img tags should be closed 
`<img src="assets/images/logo.jpeg" class="hvr-pulse-grow" />`

Reply:
done